addons/website_sale/static/src/js/product_configurator_dialog/product_configurator_dialog.js

```diff
@@ -14,6 +14,8 @@ patch(ProductConfiguratorDialog, {
             shape: {
                 ...ProductConfiguratorDialog.props.options.shape,
                 isMainProductConfigurable: { type: Boolean, optional: true },
+                showQuantity: { type: Boolean, optional: true },
+                showPrice: { type: Boolean, optional: true },
```

Comment:
Those are already declared in `sale`, so no need to declare them again in eCommerce :)